index.d.ts

```diff
@@ -1,4 +1,13 @@
 declare namespace activeWin {
+	interface Options {
+		/**
+		Disable the screen recording permission check (macOS)
+
+		Setting this to true will prevent the Screen Recording permission prompt on macOS versions 10.15 and newer. The `title` property will always be set to an empty string.
```

Comment:
Should mention the default value with `@default`.

https://github.com/sindresorhus/typescript-definition-style-guide